packages/eslint/src/generators/workspace-rules-project/__snapshots__/workspace-rules-project.spec.ts.snap

```diff
@@ -87,9 +88,6 @@ export default {
   },
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageDirectory: '../../coverage/tools/eslint-rules',
-  moduleNameMapper: {
-    '@eslint/eslintrc': '@eslint/eslintrc/dist/eslintrc-universal.cjs',
```

Comment:
We will need a migration for this too if it is no longer something we want to apply? (I'm afraid I can't remember why this was needed originally)